src/components/TransactionsList.js

```diff
@@ -1,6 +1,10 @@
 import React from 'react'
+import Transaction from './Transaction'
 
-const TransactionsList = () => {
+
+const TransactionsList = (props) => {
+
+let filterList = props.transactions.filter((transaction)=> transaction.description.toLowerCase().includes(props.searchTerm) || transaction.category.toLowerCase().includes(props.searchTerm))
```

Comment:
I would consider doing this filtering on the `Container` class. Then, you can separate it into a method. Your TransactionList can then be a 'dumber' component - it just needs to know what list to display.